README.md

```diff
@@ -120,6 +120,21 @@ SpiceClient client = SpiceClient.builder()
 Retries are performed for connection and system internal errors. It is the SDK user's responsibility to properly
 handle other errors, for example RESOURCE_EXHAUSTED (HTTP 429).
 
+### Spice.ai Runtime commands
```

Comment:
This is currently supported only with [Spice.ai OSS](https://github.com/spiceai/spiceai) / can't be used with SCP. Removed originally added notice below as gRPC implementation is coming soon and will support both OSS and SCP 

`Currently supported only with locally running [Spice.ai OSS](https://github.com/spiceai/spiceai).`